### ISSUE TITLE
[bug] /navtype/EnumCustomNavTypes.kt Conflicting import, imported name is ambiguous

### ISSUE THREAD

**Issue by vladsakun** [raamcosta/compose-destinations#518]
I want to pass an enum to Compose Destination. I have two enum classes named `Triggered` but in two different tracking classes.
`Tracking classes:`

1️⃣
```
data class ListTracking(
    val test: String = "test",
) {

    enum class Triggered(val value: String) {
        JUST("JUST"),
        DO("DO"),
        IT("IT");
    }
}
```

2️⃣
```
data class DetailsTracking(
    val test: String = "test",
) {

    enum class Triggered(val value: String) {
        TEST1("Screen2TrackingJUST"),
        ALOHA("Screen2TrackingDO"),
        BANANA("Screen2TrackingIT");
    }
}
```

`Composable destinations:`

1️⃣
```
@RootNavGraph
@Destination
@Composable
fun ListScreen(
    triggered: ListTracking.Triggered,
) {
    Text("Aloha, this is list screen")
}
```

2️⃣

```
@RootNavGraph
@Destination
@Composable
fun DetailsScreen(
    triggered: DetailsTracking.Triggered,
) {
    Text("Aloha, this is DetailsScreen")
}
```

When I build the project I receive the following exception:
`navtype/EnumCustomNavTypes.kt:7:59 Conflicting import, imported name 'Triggered' is ambiguous`

I can't change the name of enums, because in a real world project they are dictated by business and code generation library 😭

May you provide a fix for this or any known workaround to make it build 🙏

Steps to reproduce:
1️⃣ Clone the repo: https://github.com/vladsakun/ComposeDestinationsBug
2️⃣ Build the project

**Comment by raamcosta** [comment]
Hi there 👋 

A workaround is to pass a string in one of them and then do example `DetailsTracking.Triggered.valueOf(stringArgument)` to get the actual enum entry.

I'll look into this when I have some time.

**Comment by raamcosta** [comment]
So sorry that it took me so long to fix this. It will be fixed on next releases for both compose 1.7 related releases and for v2.

V1 compose 1.6 is likely not going to get any release, in which case, this will be left as is in that version.
